### ISSUE TITLE
LRS list gone and registration is required after upgrade from 1.8 to 1.10

### ISSUE THREAD

**Issue by salexfo** [LearningLocker/learninglocker#732]
Hi,
After I've upgraded from 1.8 to 1.10 (I've adhered to the upgrade rules that apply to 1.9)
I was required to register and under the admin Dashboard everything was gone.

The upgrade was done via composer according to the steps written done here:
http://docs.learninglocker.net/upgrading/

Can someone shed light on the issue?

Thanks in advance

**Comment by daniel-abbey** [comment]
Hi @salexfo,
Sorry to hear about your problem.
Did you register using the same email address you were using in 1.8?
Kind regards,
Daniel

**Comment by salexfo** [comment]
Yep

**Comment by daniel-abbey** [comment]
Does your LRS and its statements still exist in MongoDB?

**Comment by salexfo** [comment]
I don't think so. How can I verify it? What is the collection's name?

**Comment by daniel-abbey** [comment]
The default is "ll_staging"

**Comment by davidpesce** [comment]
I've had this issue when not setting the environment. Make sure you have local/production env settings set correctly.

**Comment by ryansmith94** [comment]
".env.local.php" file and "app/config/local" directory need to be checked.

**Comment by salexfo** [comment]
Hi,
The statements are there. When upgrading I've migrated the "local" environment.

**Comment by daniel-abbey** [comment]
@salexfo, I have tried to reproduce your error using the following steps:

1. Install a fresh copy of LL following the [documented installation process](http://docs.learninglocker.net/installation/) using the git clone method.
2. Checkout v1.8.0
3. POST some dummy statements and verify their creation via LRS dashboard and mongo
4. Upgrade to v1.10.0 following the [documented upgrade procedure](http://docs.learninglocker.net/upgrading/)
5. Verify that the LRS and statements still exist

Unfortunately I did not run into any problems and cannot confirm your issue.
